### ISSUE TITLE
can't repeat single song

### ISSUE THREAD

**Issue by Canorus** [BLeeEZ/amperfy#41]
- iOS 14.4.2
- Amperfy 1.0.4
- Ampache Server API version 441000
- Client API Version 350001

can't repeat single song. song automatically stops.

**Comment by BLeeEZ** [comment]
You are right. I haven't used the repeat song a while ago. I will fix it asap.

**Comment by BLeeEZ** [comment]
This bug is fixed with this commit 6f1e7ba8984c2c5a0d0df9662a304d86a3ba2429.

**Comment by Canorus** [comment]
thank you very much. I was bit worried if it's just my device acting up.

**Comment by sandersantema** [comment]
Is this fixed? Using Ampache 1.0.5 (3) and setting the repeat symbol to `1` the song simply skips to the next instead of repeating.

**Comment by sandersantema** [comment]
But shouldn't setting the repeating symbol to one then keep repeating the same song?

**Comment by Canorus** [comment]
As I can confirm that this issue has been resolved in 1.0.5(5), I'm closing the issue. Thank you so much for your hard work and thank you for making such a great app.
